support.sh

```diff
@@ -18,6 +18,24 @@ do_sed () {
 	fi
 }
 
+# Reverse the file from filename or stdin
+do_tac () {
+	if [ "$(uname)" == "Darwin" ]; then # Mac
+		# macOS doesn't have tac, so we use tail -r
```

Comment:
Doesn't Linux have tail? So we could just use tail everywhere?

Reply:
> Doesn't Linux have tail? So we could just use tail everywhere?

There is no -r option on linux (checked on ubuntu): 
```
root@fe890b91bb61:/# tail -r test.txt
tail: invalid option -- 'r'
Try 'tail --help' for more information.
```

Reply:
Okay, that’s fine. Can you update the comment then to explain that macOS doesn’t have tac and Linux doesn’t have `tail -r`, hence the two approaches. Thanks for testing both.

Also please fix the whitespace on many of these lines; this repo uses tabs.

Reply:
done